src/Parsley/Schema/Blocks.php

```diff
@@ -75,13 +75,19 @@ public function marks(): array
             [
                 'tag' => 'a',
                 'attrs' => ['href', 'target', 'title'],
+                'defaults' => [
+                    'rel' => 'noopener noreferrer nofollow'
+                ]
```

Comment:
Does this line have anything to do with fixing this issue?

https://github.com/getkirby/kirby/issues/3736#issuecomment-931592854

Reply:
Ah, damn. That was just a test. I wanted to remove it again. We need to solve it differently. Thanks for finding it.